Code/GraphMol/Atropisomers.cpp

```diff
@@ -554,13 +553,7 @@ void detectAtropisomerChirality(ROMol &mol, const Conformer *conf) {
       continue;
     }
 
-    if (DetectAtropisomerChiralityOneBond(bondToTry, mol, conf)) {
-      foundAtrop = true;
-    }
-  }
-
-  if (foundAtrop) {
-    cleanupAtropisomerStereoGroups(mol);
+    DetectAtropisomerChiralityOneBond(bondToTry, mol, conf);
```

Comment:
I don't understand what this call is doing if the return value is unused. It looks like it doesn't stores any data anywhere.

Reply:
yeah, the return value is not being used any more. I will change the function to return void